Sources/ILToJS/Arguments.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
```

Comment:
I guess we shouldn't duplicate this file. Maybe just move it into Fuzzilli/Util?